src/constants.ts

```diff
@@ -3,240 +3,241 @@
+declare const CREEP_CORPSE_RATE: 0.2;
 
 declare const OBSTACLE_OBJECT_TYPES: string[];
 
-declare const ENERGY_REGEN_TIME: number;
-declare const ENERGY_DECAY: number;
+declare const ENERGY_REGEN_TIME: 300;
+declare const ENERGY_DECAY: 1000;
 
-declare const REPAIR_COST: number;
+declare const REPAIR_COST: 0.01;
 
-declare const RAMPART_DECAY_AMOUNT: number;
-declare const RAMPART_DECAY_TIME: number;
-declare const RAMPART_HITS: number;
+declare const RAMPART_DECAY_AMOUNT: 300;
+declare const RAMPART_DECAY_TIME: 100;
+declare const RAMPART_HITS: 1;
 declare const RAMPART_HITS_MAX: {
-    2: number,
-    3: number,
-    4: number,
-    5: number,
-    6: number,
-    7: number,
-    8: number
+    2: 300000,
+    3: 1000000,
+    4: 3000000,
+    5: 10000000,
+    6: 30000000,
+    7: 100000000,
+    8: 300000000
 };
 
-declare const SPAWN_HITS: number;
-declare const SPAWN_ENERGY_START: number;
-declare const SPAWN_ENERGY_CAPACITY: number;
+declare const SPAWN_HITS: 5000;
+declare const SPAWN_ENERGY_START: 300;
+declare const SPAWN_ENERGY_CAPACITY: 300;
 
-declare const SOURCE_ENERGY_CAPACITY: number;
-declare const SOURCE_ENERGY_NEUTRAL_CAPACITY: number;
-declare const SOURCE_ENERGY_KEEPER_CAPACITY: number;
+declare const SOURCE_ENERGY_CAPACITY: 3000;
+declare const SOURCE_ENERGY_NEUTRAL_CAPACITY: 1500;
+declare const SOURCE_ENERGY_KEEPER_CAPACITY: 4000;
 
-declare const WALL_HITS: number;
-declare const WALL_HITS_MAX: number;
+declare const WALL_HITS: 1;
+declare const WALL_HITS_MAX: 300000000;
 
-declare const EXTENSION_HITS: number;
-declare const EXTENSION_ENERGY_CAPACITY: number;
+declare const EXTENSION_HITS: 1000;
+declare const EXTENSION_ENERGY_CAPACITY: {0: 50, 1: 50, 2: 50, 3: 50, 4: 50, 5: 50, 6: 50, 7: 100, 8: 200};
 
-declare const ROAD_HITS: number;
-declare const ROAD_WEAROUT: number;
-declare const ROAD_DECAY_AMOUNT: number;
-declare const ROAD_DECAY_TIME: number;
+declare const ROAD_HITS: 5000;
+declare const ROAD_WEAROUT: 1;
+declare const ROAD_DECAY_AMOUNT: 100;
+declare const ROAD_DECAY_TIME: 1000;
 
-declare const LINK_HITS: number;
-declare const LINK_HITS_MAX: number;
-declare const LINK_CAPACITY: number;
-declare const LINK_COOLDOWN: number;
-declare const LINK_LOSS_RATION: number;
+declare const LINK_HITS: 1000;
+declare const LINK_HITS_MAX: 1000;
+declare const LINK_CAPACITY: 800;
+declare const LINK_COOLDOWN: 1;
+declare const LINK_LOSS_RATIO: 0.03;
 
-declare const STORAGE_CAPACITY: number;
-declare const STORAGE_HITS: number;
+declare const STORAGE_CAPACITY: 1000000;
+declare const STORAGE_HITS: 10000;
 
 declare const BODYPART_COST: {
     [part: string]: number;
-    move: number;
-    work: number;
-    attack: number;
-    carry: number;
-    heal: number;
-    ranged_attack: number;
-    tough: number;
-    claim: number;
+    move: 50;
+    work: 100;
+    attack: 80;
+    carry: 50;
+    heal: 250;
+    ranged_attack: 150;
+    tough: 10;
+    claim: 600;
 };
 declare const BODYPARTS_ALL: string[];
```

Comment:
I'll see about adding this one in with my issue-#91 changes, which should present a unified type for many of these items.